Doc/c-api/long.rst

```diff
@@ -464,3 +464,10 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
 
    Otherwise, the return value is undefined.
 
+.. c:function:: int PyLong_FileDescriptor_Converter(PyObject *p, void *ptr)
+
+   Convert a Python object to a file descript (C ``int`` type).
+   Set *\*ptr* and return ``1`` on success. Set an exception and return ``0`` on error.
```

Comment:
Style nit (feel free to ignore); I prefer semantic line-breaks for new docs :)

```suggestion
   Set *\*ptr* and return ``1`` on success.
   Set an exception and return ``0`` on error.
```

Reply:
Rendered as HTML, it's the same, no? I applied your suggestion ☺️

Reply:
Yes, it renders the same, but the diff is nicer if we need to amend one of the sentences in the future ;)